channels.go

```diff
@@ -0,0 +1,134 @@
+package telebot
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+// ChatInfo struct is the main chat information from telegram
```

Comment:
The official API docs state "This object represents a chat." so a better description would likely be "ChatInfo represents a chat".